sample/Sample.jsx

```diff
@@ -1,9 +1,51 @@
 import React, { useState } from 'react';
 import { useSetInterval } from '@wojtekmaj/react-hooks';
-import Clock from '@viviedu/react-clock';
+import Clock from '../src/Clock';
 
+import '../src/Clock.less';
 import './Sample.less';
 
+function OldClock(props) {
+  return (
+    <Clock
+      hourHandWidth={5}
+      minuteHandWidth={3}
+      renderNumbers={true}
+      secondHandLength={70}
+      secondHandWidth={2}
+      size={500}
+      {...props}
+    />
+  );
+}
+
+function NewClock(props) {
+  return (
+    <Clock
+      hourHandLength={30}
+      hourHandWidth={8}
+      hourMarksLength={30}
+      hourMarksWidth={4}
+      hourHandOppositeLength={0}
+      minuteHandLength={50}
+      minuteHandWidth={7}
+      minuteHandOppositeLength={0}
+      newUI={true}
+      renderMinuteMarks={false}
+      renderNumbers={true}
+      secondHandLength={88}
+      secondHandWidth={7}
+      size={500}
+      {...props}
+    />
+  );
+}
+
+function ClockWrapper({ newUI, ...props }) {
+  return newUI ? <NewClock {...props} /> : <OldClock {...props} />;
```

Comment:
What's the point of this? Do we want non-multizone widgets to be old style still?

Reply:
Also no need for semicolon here

Reply:
This is the Sample app, for dev testing only